tests/testcontainers/test_h2spec.rs

```diff
@@ -305,8 +361,12 @@ fn label_log(label: &'static str, stream: impl futures::Stream<Item=Result<LogOu
         .map(move |line| format!("{}: {}", label, line));
 }
 
-async fn get_container_exit_code(docker: &Docker, container: &ContainerCreateResponse) -> Option<i64> {
-    let final_inspect = docker.inspect_container(&*container.id, Some(InspectContainerOptions { size: false })).await.unwrap();
-    let exit_code = final_inspect.state.unwrap().exit_code;
+async fn get_container_exit_code(docker: &Docker, container: &ContainerCreateResponse) -> Result<i64, Box<dyn std::error::Error>> {
```

Comment:
see https://doc.rust-lang.org/rust-by-example/error/multiple_error_types/boxing_errors.html for justification of the return type